awards_and_contracts.md: update purchase orders example (#1720)
open-contracting/standard: docs/guidance/map/awards_contracts.md

[View the example in Paraguay's API](https://contrataciones.gov.py/datos/api/v3/doc/ocds/record/ocds-03ad3f-340885-1)
```

## Purchase orders

A purchase order is a specific type of contract, an official document issued by a buyer committing to pay a supplier for the supply of specific goods, services or works to be delivered in the future.

Purchase orders can be issued against an existing contract, or if no prior contract exists then acceptance of a purchase order by a supplier forms a contract between buyer and supplier.

Purchase orders that are made against contracts with a definite quantity or value of items ought to not be disclosed in the `contracts` section of OCDS, due to the risk of double counting items on the purchase order and the contract it relates to. Instead they ought to be disclosed in the `implementation.transactions` array of the `Contract`.

### Example: Purchase orders

> The Bureau of Parks and Recreation in Portland, Oregon, USA, awards a 2 year contract worth up to $1m to a local contractor to maintain Mt. Tabor Park. This contract specifies the goods, services and works to be provided along with unit prices and estimates of quantities. The contract is represented in the `contracts` section of OCDS as follows:

Reviewer: Copy-edit to remove information that isn't relevant to what the example is intended to illustrate.

```suggestion
A buyer awards a contract for park maintenance services with a maximum value of $1 million to a single supplier. The contract specifies the services, unit prices, and estimated quantities. It is added to the `contracts` array.
```

Author: no longer relevant as the example has been completed changed

```{csv-table-no-translate}
:header-rows: 1
:file: ../../examples/purchase_orders/parent_contract.csv
```

> The contract in this scenario is treated as a type of [framework agreement](framework_agreements.md), with the contract value being disclosed in the `maximumValue` field instead of the `value` field.

Reviewer: Edit: Moved to https://github.com/open-contracting/standard/issues/1694#issuecomment-2492546747.

I haven't reviewed beyond this point.

Purchase orders can be issued against an existing contract, or if no prior contract exists then acceptance of a purchase order by a supplier forms a contract between buyer and supplier.

> Purchase orders can be made against contracts with indefinite quantities or maximum only values. In these cases the initial contract and resulting purchase orders ought to be modelled as [a closed framework agreement with no second-stage competition](https://standard.open-contracting.org/staging/1.2-dev/en/guidance/map/framework_agreements/).

Reviewer: ```suggestion
Purchase orders can be made against contracts that specify only minimum and maximum limits on the quantity or value of the items being procured. Such contracts are known as indefinite-delivery/indefinite-quantity (IDIQ) contracts. IDIQ contracts are considered to be two-stage procurement techniques and ought to be modelled according to the [guidance for framework agreements](https://standard.open-contracting.org/staging/1.2-dev/en/guidance/map/framework_agreements/).
```

> Purchase orders that are made against contracts with a definite quantity or value of items ought to not be disclosed in the `contracts` section of OCDS, due to the risk of double counting items on the purchase order and the contract it relates to.

Reviewer: ```suggestion
Purchase orders made against contracts with definite quantities or values ought to not be disclosed in the `contracts` section of OCDS, due to the risk of double counting.
```

### Example: Double counting contracts and purchase orders

> Paraguay's Dirección Nacional de Ingresos Tributarios awards a contract worth 3,150 million Gs to Carlos Ruben Oviedo Centurion for cleaning services in administrative buildings in Asunción. The contract is represented in the `contracts` section of OCDS as follows:

Reviewer: ```suggestion
A buyer awards a contract worth 3,150m Gs for cleaning services across several office buildings. The contract is added to the `contracts` array.
```

```{csv-table-no-translate}
:header-rows: 1
:file: ../../examples/purchase_orders/parent_contract.csv
```

> Calculating the sum of the contract value in the above example gives the correct result of Gs 3,150 million.

Reviewer: ```suggestion
```

> During the first month of the contract Dirección Nacional de Ingresos Tributarios issues 3 purchase orders for cleaning in 3 different buildings.

Reviewer: ```suggestion
The buyer issues three purchase orders for cleaning in three different buildings.
```

> If purchase orders were also disclosed in the `contracts` section of OCDS, the `contracts` section of OCDS would now be populated as follows:

Reviewer: ```suggestion
If the purchase orders were added to the `contracts` array, it would lead to double counting. The total value of the contracts in the OCDS data would be greater than the value of the real-life contract. 
```

```{csv-table-no-translate}
:header-rows: 1
:file: ../../examples/purchase_orders/contracts_pos.csv
```

> Calculating the sum of the contract value in the above example gives an incorrect result of Gs 3,211 million.

Reviewer: ```suggestion
The correct approach is to add only the 'parent' contract to the `contracts` array.
```

```{note}

> The approach for modelling purchase orders in OCDS has not been standardized. Implementers are encouraged to describe their use of purchase orders in this ([GitHub issue](https://github.com/open-contracting/standard/issues/897)) to help develop a standardized approach.

Reviewer: ```suggestion
The approach to modelling purchase orders in OCDS is not yet standardized. Implementers are encouraged to describe their use of purchase orders in the ([GitHub issue](https://github.com/open-contracting/standard/issues/897)) to help develop a standardized approach.
```
